java/test/jmri/jmrit/logixng/SocketOperationTest.java

```diff
@@ -20,19 +20,19 @@ public class SocketOperationTest {
 
     @Test
     public void testAddRemoveChildren() throws PropertyVetoException, Exception {
-        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
```

Comment:
Can I recommend that you keep the isHeadless check?  CI separate tests are run with a screen, but I sometimes do a standard `ant headlesstest` that doesn't do the skip-tests.

Reply:
> Can I recommend that you keep the isHeadless check? CI separate tests are run with a screen, but I sometimes do a standard `ant headlesstest` that doesn't do the skip-tests.

Done.